FinalProject/FinalProject/src/main/java/com/globant/FinalProject/controller/ShoppingController.java

```diff
@@ -0,0 +1,100 @@
+package com.globant.FinalProject.controller;
+
+import java.sql.SQLException;
+import java.util.ArrayList;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+import com.globant.FinalProject.DTO.DTOBuy;
+import com.globant.FinalProject.shopping.ManagerCart;
+import com.globant.FinalProject.shopping.Product;
+import com.globant.FinalProject.shopping.ShoppingCart;
+import com.globant.FinalProject.user.Registration;
+import com.globant.FinalProject.user.User;
+import com.globant.FinalProject.user.UserRepository;
+
+/**
+ * This controller handles requests for the shopping center
+ * @author Herminio Andres Hernandez
+ * @version 05/08/2018
+ */
+@Controller
+@RequestMapping("/api")
+public class ShoppingController {
+
+	private User currentUser = null;
```

Comment:
this will allow only one user at the same time, also if I know the user id i'll be able to access it's data once he's logged.